### ISSUE TITLE
[www] Showcase filter not responding to forward/backward buttons

### ISSUE THREAD

**Issue by universse** [gatsbyjs/gatsby#11733]
Hi. The filters on https://www.gatsbyjs.org/showcase aren't responding to forward/backward navigation buttons. There seems to be a bug in components/url-query.

Should be

```
this.unlisten = globalHistory.listen(({ location: { search }) => {
      this.getDerivedStateFromQuery(search)
})
```

instead of

```
this.unlisten = globalHistory.listen(({ search }) => {
      this.getDerivedStateFromQuery(search)
})
```

**Comment by danedavid** [comment]
@universse There doesn't seem to be a trouble with the site showcase modal previous & next buttons.

**Comment by universse** [comment]
Sorry, guess I wasn't clear enough. When you click on sidebar filters both the URL and the Sites section will update. When you click the browser back/forward buttons (not the modal) after clicking a few sidebar filters, the URL will update but the Sites section does not.

**Comment by danedavid** [comment]
Guess this has been fixed in #11770. Hence closing.
